src/scss/_app_styling.scss

```diff
@@ -37,7 +37,7 @@ $app-light-grey: #f8f8f8;
   }
 }
 
-.app-whats-new {
+.app-light-grey {
```

Comment:
Question - is this how we use govuk styling in express of is this a custom styling.

Reply:
This is custom styling and not GDS styling

Reply:
Why don't we use GDS styling?